README.md

```diff
@@ -29,6 +29,28 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Adding Throttling

```

Comment:
## Throttling

Reply:
In out opinion `CPU Throttling` seem a more appropiate title.